Introduction/Readme.md

```diff
@@ -0,0 +1,23 @@
+What are the three most essential skills that a beginner programmer needs to know?
+
+Please explain these three skills in your own words.
+
+Why do you need to not copy and paste?
+
+Why is Practice and Persistence important?
+
+
+
+< Problem Solving, Critical Thinking, Debugging>
```

Comment:
Please use the three essential skills as discussed in the http://ruby.learncodethehardway.org/book/intro.html chapter.